src/Model/Event.php

```diff
@@ -257,7 +257,7 @@ public static function store($arr)
 		$event['deny_cid']  =        $arr['deny_cid']  ?? '';
 		$event['deny_gid']  =        $arr['deny_gid']  ?? '';
 		$event['adjust']    = intval($arr['adjust']    ?? 0);
-		$event['nofinish']  = intval(!empty($arr['nofinish'] || !empty($event['start']) && empty($event['finish'])));
```

Comment:
This should probably have been 
```
$event['nofinish']  = intval(!empty($arr['nofinish']) || !empty($event['start']) && empty($event['finish']));
```
Notice the closing parenthesis just after the `$arr['nofinish']`.

Your solution works too.

Reply:
Yeah, I had it this way at first. But the second solutions is nearer to the previous situation, where the second part had been used only when the first part hadn't existed.

Reply:
Fair enough.